data/pilots/rebel-alliance/t-65-x-wing.json

```diff
@@ -307,6 +307,262 @@
       "extended": true,
       "keywords": ["X-wing"],
       "epic": true
+    },
+    {
+      "name": "Biggs Darklighter",
+      "caption": "Battle of Yavin",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 5,
+      "standardLoadout": true,
+      "xws": "biggsdarklighter-battleofyavin",
+      "ability": "During the System Phase, you may choose 1 friendly ship at range 1. If you do, treat your initiative as equal to the chosen ship's initiative until the end of the Activation Phase.",
+      "shipAbility": {
+        "name": "Hope",
+        "text": "After another friendly ship at range 0-3 is destroyed, you may perform a [Focus] or [Boost] action."
+      },
+      "upgrades": [
```

Comment:
The "upgrades" name is totally new. Any issue with upgrades? I couldn't think of anything better.